probe_scraper/scrapers/moz_central_scraper.py

```diff
@@ -251,56 +249,6 @@ def save_error_cache(folder, error_cache):
         json.dump(error_cache, f, sort_keys=True, indent=2, separators=(',', ': '))
 
 
-def scrape_release_tags(folder=None, min_fx_version=None, max_fx_version=None, channels=None):
-    """
-    Returns data in the format:
-    {
-      <channel>: {
-        <revision>: {
-          "channel": <channel>,
-          "version": <major-version>,
-          "registries": {
-            "event": [<path>, ...],
-            "histogram": [<path>, ...],
-            "scalar": [<path>, ...]
-          }
-        },
-        ...
-      },
-      ...
-    }
-    """
-    if min_fx_version is None:
-        min_fx_version = MIN_FIREFOX_VERSION
-    if folder is None:
-        folder = tempfile.mkdtemp()
-
-    error_cache = load_error_cache(folder)
-    requests_cache.install_cache("probe_scraper_cache")
-    results = defaultdict(dict)
-
-    if channels is None:
-        channels = CHANNELS.keys()
-
-    for channel in channels:
-        tags = load_tags(channel)
-        versions = extract_tag_data(tags, channel, min_fx_version, max_fx_version)
```

Comment:
Good news! We can remove _everything_ related to using json-tags above.

Reply:
Oops my code inspection didn't highlight the functions as unused